### PR TITLE
brie: add integration test for br

### PR DESCRIPTION
**Pull request by 3pointer**
### What problem does this PR solve?

Problem Summary: add BR integration for TiDB

### What is changed and how it works?

What's Changed:
1. BR only work on tikv store. so add a test in session_test which has tikv store.
2. Since https://github.com/pingcap/br/pull/443 handle all BR related configurations internally, we can remove useless setting config code

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Integration test

### Release note 

- No release note

### CODE REVIEW

File: session/session_test.go [hunk 1]
```diff
@@ -3219,3 +3225,34 @@ func (s *testSessionSuite2) TestPerStmtTaskID(c *C) {
...
+		tk.MustExec("create database if not exists br")
+		tk.MustExec("use br")
+		tk.MustExec("create table t1(v int)")
+		tk.MustExec("insert into t1 values (1)")
+		tk.MustExec("insert into t1 values (2)")
+		tk.MustExec("insert into t1 values (3)")
+
+		tk.MustQuery("select count(*) from t1").Check(testkit.Rows("3"))
+
+		os.RemoveAll("/tmp/bk1")
```
Comment (tiancaiamao) [inline session/session_test.go:3246]: How about using API like [os.TempDir](https://pkg.go.dev/os?tab=doc#TempDir) and [path.Join](https://pkg.go.dev/path?tab=doc#Join) to make the code more general?

Comment (3pointer) [reply session/session_test.go]: good idea

**Comment by tiancaiamao** [comment]
LGTM

**Comment by glorv** [comment]
lgtm

**Comment by overvenus** [comment]
LGTM

**Pull request merged**